doc/vfd-swmr-user-guide.md

```diff
@@ -0,0 +1,496 @@
+# Welcome to VFD SWMR  
+
+Thank you for volunteering to test VFD SWMR.
```

Comment:
This file needs a sanity check - there's a number of hypotheticals and future-looking statements that may no longer be valid.

Reply:
This is not the final user guide and will be removed before the merge. The real VFD SWMR documentation will be added to the doxygen UG.